awx/api/templates/api/base_variable_data.md

```diff
@@ -1,9 +1,13 @@
+{% ifmeth GET %}
```

Comment:
This is a new template tag we can use to compose Swagger docs from our existing API browser Markdown.  The idea is that as `django-rest-swagger` builds the API descriptions from this markdown, it does so for _each_ potential request method.  So we'll generate this view for (potentially) `GET`, `PUT`, `POST`, `DELETE`.

This `ifmeth` tag allows you to provide hints to the template rendering to filter out blocks that aren't applicable to the current method it's processing.